RecoBTag/Combined/src/hiRun2LegacyCSVv2Tagger.cc

```diff
@@ -0,0 +1,175 @@
+#include "RecoBTag/Combined/interface/hiRun2LegacyCSVv2Tagger.h"
+
+#include "DataFormats/BTauReco/interface/CandSoftLeptonTagInfo.h"
+#include "DataFormats/BTauReco/interface/CandIPTagInfo.h"
+#include "DataFormats/BTauReco/interface/SecondaryVertexTagInfo.h"
+#include "FWCore/Utilities/interface/ESInputTag.h"
+#include "CondFormats/DataRecord/interface/GBRWrapperRcd.h"
+
+#include <iostream>
+#include <memory>
+
+#include <algorithm>
+#include <iostream>
+#include <map>
+#include <vector>
+
+const bool PbPbdebug = false;
+
+hiRun2LegacyCSVv2Tagger::Tokens::Tokens(const edm::ParameterSet &configuration, edm::ESConsumesCollector &&cc) {
+  if (configuration.getParameter<bool>("useCondDB")) {
+    gbrForest_ = cc.consumes(edm::ESInputTag{"",
+                                             configuration.existsAs<std::string>("gbrForestLabel")
+                                                 ? configuration.getParameter<std::string>("gbrForestLabel")
+                                                 : ""});
+  }
+}
+
+hiRun2LegacyCSVv2Tagger::hiRun2LegacyCSVv2Tagger(const edm::ParameterSet &configuration, Tokens tokens)
+    : sl_computer_(configuration.getParameter<edm::ParameterSet>("slComputerCfg")),
+      sv_computer_(configuration.getParameter<edm::ParameterSet>("slComputerCfg")),
```

Comment:
remove the sl_computer_ and update the config name
```suggestion
    : sv_computer_(configuration.getParameter<edm::ParameterSet>("svComputerCfg")),
```